index.html

```diff
@@ -1,38 +1,196 @@
+      </p>
+    </div>
+  </header>
 
-<section>
+  <section class="main_program">
+    <div class="main_program_content"></div>
+    <div class="head">
+      <h2>Main Program</h2>
+      <hr class="slim" />
+    </div>
+    <div class="content-body">
+      <div class="content">
+        <span><i class="fas fa-podium-star fa-2x"></i></span>
+        <h3>Lecture</h3>
+        <p>
+          Listen to the speakers from various countries about the insight on
+          the future of drone taxi
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fas fa-drone-alt fa-2x"></i></span>
+        <h3>Drone Exhibition</h3>
+        <p>
+          Apprieciate various creation and innovative inventions, Orginised
+          from the Art Centre Cairo
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fad fa-comments-alt fa-2x"></i></span>
+        <h3>Forum</h3>
+        <p>
+          Have time to share your thoughts and opinions with experts on each
+          topic
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fas fa-chalkboard-teacher fa-2x"></i></span>
+        <h3>Workshop</h3>
+        <p>
+          Try to create your own working Drone using high-end technology
+          rather that just watching
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fad fa-project-diagram fa-2x"></i></span>
+        <h3>Collaborate</h3>
+        <p>
+          Get opportunities to network with top drone engineers from all over
+          the world, also after the summit
+        </p>
+      </div>
 
-</section>
+    </div>
+    <div class="button">
+      <button type="submit">
+        Join Drones and Drones Taxi Global Summit
+      </button>
+    </div>
+  </section>
 
+  <section class="feature_speakers flex-column-center">
+    <div class="speaker_background"></div>
+    <div class="feature-container">
+      <h2>Featured Speakers</h2>
+      <hr class="slim">
+    </div>
+    <ul class="feature-list">
 
-  <script src="./js/object.js"></script>
-  <script src="./js/script.js"></script>
+    </ul>
+  </section>
+
+  <section class="partners">
+    <h2>Partner</h2>
+    <hr class="slim">
+    <ul>
+      <li><i class="fab fa-firefox fa-3x"></i></li>
+      <li><i class="fab fa-google fa-3x"></i></li>
+      <li><i class="fab fa-apple fa-3x"></i></li>
+      <li><i class="fab fa-bitcoin fa-3x"></i></li>
+      <li><i class="fab fa-airbnb fa-3x"></i></li>
+    </ul>
+  </section>
+  <footer>
+    <div class="copyright">
+      <div class="coperation">
+        <img src="./asset/image/logo_mobile.png" alt="logo-footer">
+      </div>
+      <div class="rights reserve">
+        <p>2021 Drones and Drone Taxi Some Rights Reserved</p>
+        <br>
+        <p class="arabic">انضم إلينا في القمة العالمية للـ دي.دي.تي ونأمل أن نراك هناك</p>
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] **OPTIONAL**
You managed the right alignments to the elements of the footer. However, it could use a little bit of improvement for better looking 🙏 😄

>A screenshot from your project.
![Screenshot3](https://user-images.githubusercontent.com/47625407/124309835-a1f36a80-db6b-11eb-9799-2ad6d2473100.png)

_**Please refer to the review summary and the given design link for more information.**_